src/SFA.DAS.ApplyService.InternalApi/AutoMapper/UkrlpCharityCommissionProfile.cs

```diff
@@ -0,0 +1,31 @@
+﻿using AutoMapper;
+using CharityCommissionService;
+using SFA.DAS.ApplyService.Domain.CharityCommission;
+using SFA.DAS.ApplyService.Domain.CompaniesHouse;
+using System.Linq;
+namespace SFA.DAS.ApplyService.InternalApi.AutoMapper
+{
+    public class UkrlpCharityCommissionProfile : Profile
+    {
+        public UkrlpCharityCommissionProfile()
+        {
+            CreateMap<InternalApi.Types.CharityCommission.Charity, CharityCommissionSummary>()
+               .ForMember(dest => dest.CharityName, opt => opt.MapFrom(source => source.Name))
+               .ForMember(dest => dest.CharityNumber, opt => opt.MapFrom(source => source.CharityNumber))
+               .ForMember(dest => dest.IncorporatedOn, opt => opt.MapFrom(source => source.IncorporatedOn))
+               .ForMember(dest => dest.Trustees, opt => opt.MapFrom(source => source.Trustees))
+               .ForMember(dest => dest.TrusteeManualEntryRequired, opt => opt.MapFrom(source => source.TrusteeManualEntryRequired))
+               .ForAllOtherMembers(opt => opt.Ignore());
+        }
+    }
+    public class CharityTrusteeProfile : Profile
```

Comment:
Multiple classes in a file.  It seems to be a common approach but can we stop please? :)